io_scene_nif/modules/armature/__init__.py

```diff
@@ -106,7 +107,54 @@ def get_bone_name_for_nif(name):
         return name
     return name
 
+def set_bone_correction_from_version(version):
+    if version in (0x14020007, ):
```

Comment:
We have to check other game versions for their axis orientation so the correction axes can be set accordingly.

Reply:
Yeah, but less issues being reported. 
Though I think the alternative of the user selecting the correct option first time, taking the time to retry with a different option, read the docs.... 😝